src/lib.rs

```diff
@@ -92,15 +96,16 @@ mod proptests;
 #[macro_use]
 extern crate log;
 
+#[doc(inline)]
+pub use crate::kv::{codec, BoundRange, Key, KvPair, ToOwnedRange, Value};
```

Comment:
Does codec need to be public?

Reply:
For users using only this library, no. Is it possible that someone wants to manually encode keys to directly interact with PD?

Reply:
I think that might be possible, but then it should be exported by the PD crate and not the main crate.

Reply:
Makes sense.